srcpkgs/libdispatch/template

```diff
@@ -0,0 +1,32 @@
+pkgname=libdispatch
+version=5.5.3
+revision=1
+create_wrksrc=yes
+build_style=cmake
+hostmakedepends="ninja clang git"
+short_desc="Library provides comprehensive support for concurrent code execution on multicore hardware"
+maintainer="Ilya <ISPonomarev@yahoo.com>"
+license="Apache-2.0"
+homepage="https://github.com/apple/swift-corelibs-libdispatch"
+
+do_fetch() {
+       git clone https://github.com/apple/swift-corelibs-libdispatch
+       mv swift-corelibs-libdispatch ${pkgname}-${version}
+}
+
+pre_configure() {
+       mkdir build
+}
+
+do_configure() {
```

Comment:
Can't this be done with the `build_style=cmake`

Reply:
Get rid of the do_configure() and just add `configure_args="-DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++"`